addons/packages/vsphere-cpi/1.23.0-alpha.1/bundle/config/upstream/vsphere-paravirtual-cpi/03-deployment.yaml

```diff
@@ -0,0 +1,87 @@
+#@ load("/values.star", "values")
+
+#@ if values.vsphereCPI.mode == "vsphereParavirtualCPI":
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: guest-cluster-cloud-provider
+  namespace: vmware-system-cloud-provider
+  annotations:
+    cloud-provider: vsphere-paravirtual
+spec:
+  selector:
+    matchLabels:
+      name: guest-cluster-cloud-provider
+  template:
+    metadata:
+      labels:
+        name: guest-cluster-cloud-provider
+    spec:
+      hostNetwork: true
+      priorityClassName: system-cluster-critical
+      containers:
+        - args:
+            - --controllers=service
+            - --controllers=cloud-node
```

Comment:
Where is this checking been taken care of?
https://gitlab.eng.vmware.com/core-build/cayman_cloud-provider-vsphere/-/blob/vmware-1.22.4/cloud_provider_vsphere/add-on/v1.22/cloud-controller-manager.yaml#L144-153

Reply:
added a TODO. will add a field `AntreaNSXRouted` once routable pods are supported